src/Management/src/Endpoint/Services/ServiceDescriptorConverter.cs

```diff
@@ -0,0 +1,61 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Text.RegularExpressions;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.FileSystemGlobbing.Internal;
+using Steeltoe.Management.Endpoint.Hypermedia;
+
+namespace Steeltoe.Management.Endpoint.Services;
+internal class ServiceDescriptorConverter : JsonConverter<ServiceContextDescriptor>
+{
+    public override ServiceContextDescriptor Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+    {
+        throw new NotImplementedException();
+    }
+
+    public override void Write(Utf8JsonWriter writer, ServiceContextDescriptor value, JsonSerializerOptions options)
+    {
+        writer.WriteStartObject();
+        writer.WritePropertyName("contexts");
+        writer.WriteStartObject();
+     
+        foreach (var contextKey in value.Keys)
+        {
+           
+            writer.WritePropertyName(contextKey);
+            writer.WriteStartObject();
+            writer.WritePropertyName("beans");
```

Comment:
We have done this in the past by providing multiple converters and using configuration to drive the converter used as here: https://github.com/SteeltoeOSS/Steeltoe/blob/a8ba2ae9a696c508cf4b256fcc621b7d6720e294/src/Management/test/Endpoint.Test/Health/EndpointMiddlewareTest.cs#L127

Reply:
After talking to @bart-vmware seems like rather than enhance anything right now, to leave it as is and then see if the need for customizability and multiple options is still necessary.